client/src/extension.ts

```diff
@@ -156,6 +158,18 @@ export async function activate (context: vscode.ExtensionContext, launchOptions:
     client.sendNotification(jobs.Request.apiAddJar, { uri })
   })
 
+  // watch for changes to the flix.toml file
+  tomlWatcher = vscode.workspace.createFileSystemWatcher('**/flix.toml')
+  tomlWatcher.onDidChange(() => {
+    const doReload = vscode.window.showInformationMessage(
+        "Do you want to reload the flix extension after changing flix.toml?", "Reload", "No")
```

Comment:
"The flix.toml file has changed. Do you want to restart the compiler?"

Yes/No